### PR TITLE
ASC-1517 update developer docs to include junit.xsd

### PR DESCRIPTION
**Pull request by zreichert**

### CODE REVIEW

File: docs/devolper_docs.rst [hunk 1]
```diff
@@ -26,6 +26,11 @@ APIs used by ZigZag
 
 Generally Zigzag uses APIs provided by the `swagger_client`_ but there have been scenarios where bugs have prevented us from doing this.  In these cases we are using the `qTest Manager API`_ endpoints without going through the swagger client.  The library we use to do this is requests.
 
+ZigZag's xsd
+------------
+
+When ZigZag is asked to process an xml document it uses an xsd to perform the first level of validation for the document.  The file `junit.xsd`_ is used to perform validation for xml produced by any tool.  At a high level this document ensures that the xml has a <suite> element with <testcase> elements inside of it.  Logic used to validate the presence of other required attributes of the xml is handled within zigzag itself.  If piece fo information is required and it is not able te be determined with the available information is should raise an error that inherits from the base error class in `zigzag_error.py`_
```
Comment (mwbutcher) [inline docs/devolper_docs.rst]: The last two sentences should be reworded a bit:

The logic used to validate the presence of other required attributes of the xml is handled within zigzag itself.  If a piece of information is required and it is not able to be determined with the available information it should raise an error that inherits from the base error class in `zigzag_error.py`_

Comment (mwbutcher) [reply docs/devolper_docs.rst]: Other than that ^^, I'm :+1:

Comment (zreichert) [reply docs/devolper_docs.rst]: Updated

**Review by mwbutcher — APPROVED** [review]

File: docs/devolper_docs.rst [hunk 2]
```diff
@@ -26,6 +26,11 @@ APIs used by ZigZag
 
 Generally Zigzag uses APIs provided by the `swagger_client`_ but there have been scenarios where bugs have prevented us from doing this.  In these cases we are using the `qTest Manager API`_ endpoints without going through the swagger client.  The library we use to do this is requests.
 
+ZigZag's xsd
+------------
+
+When ZigZag is asked to process an xml document it uses an xsd to perform the first level of validation for the document.  The file `junit.xsd`_ is used to perform validation for xml produced by any tool.  At a high level this document ensures that the xml has a <suite> element with <testcase> elements inside of it.  The logic used to validate the presence of other required attributes of the xml is handled within zigzag itself. If a piece of information is required and it is not able to be determined with the available information it should raise an error that inherits from the base error class in `zigzag_error.py`_
```
Comment (ryan-rs) [inline docs/devolper_docs.rst]: Replace `<suite>` with `<testsuite>`.

**Review by ryan-rs — CHANGES_REQUESTED** [review]
Small typo.

**Review by ryan-rs — APPROVED** [review]

**Pull request merged**